### ISSUE TITLE
Backend ??

### ISSUE THREAD

**Issue by alissonandrade2020** [NaluFigueira/rocketshoesWeb#2]
Ele só tem front e mobile, cadê o backend da aplicação ??

**Comment by NaluFigueira** [comment]
Olá, na minha turma do bootcamp essa aplicação só tinha front end/mobile, o backend é "estático" e pode ser visto no arquivo "server.json".

**Comment by alissonandrade2020** [comment]
OK, me ajuda ??? achei muito interessante o curso de fast, já até baixei, rsrs, só que não estou conseguindo rodar ele, remumindo colocar pra funcionar ! rsrs, me ajuda ??

**Comment by NaluFigueira** [comment]
Pra rodar o front-end, basta rodar o yarn + yarn start, o mobile seria yarn + react-native run-android, e o json com o comando json-server --watch server.json

**Comment by alissonandrade2020** [comment]
![image](https://user-images.githubusercontent.com/39311340/79173019-c16a7000-7dcc-11ea-8bd0-f876b4f3dc3c.png)

**Comment by alissonandrade2020** [comment]
O front OK

**Comment by alissonandrade2020** [comment]
Me tira uma dúvida preciso fazer, yarn install ???

**Comment by NaluFigueira** [comment]
Você abriu uma issue no repositório errado, este é o do rocketshoes

**Comment by alissonandrade2020** [comment]
a tah foi mal, vou excluir

**Comment by NaluFigueira** [comment]
As instruções pra executar o fastfeet estão no readme

**Comment by alissonandrade2020** [comment]
certo, mas não estou conseguindo : (, eu abri uma Issues lá no fastfeet, ver lá prfv
